assets/src/components/notificationContent.tsx

```diff
@@ -67,6 +67,8 @@ export const title = (reason: NotificationReason): string => {
       return "CHELSEA ST BRIDGE RAISED"
     case "chelsea_st_bridge_lowered":
       return "CHELSEA ST BRIDGE LOWERED"
+    case "block_overload":
+      return "ADDED SERVICE"
```

Comment:
Random thought, but maybe we could sentence-case these and then use `text-transform: uppercase;` on the CSS end go achieve the effect we want? Not necessarily important for this ticket, especially given that we've already uppercased several other notification types here.

Reply:
We could, that would work, but I don't see a strong advantage to changing it so I'd as soon leave it as it is.

Reply:
Yeah, it's fine to leave as-is. That was just my over-active brain coming up with cool ideas that were largely orthogonal to the code review at hand. 🙂